### PR TITLE
update to tonic 0.9

### PR DESCRIPTION
**Pull request by henriiik**
Changed: Upgrade to tonic 0.9

Hello! Here comes a small PR that updates tonic to the latest version. :)

### CODE REVIEW

**Comment by YoEight** [comment]
@henriiik First of all, I'm sorry I just saw that PR!

I turned off notification on that repo by mistake.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 - No longer transitively depend on `time` 0.1  [EventStoreDB-Client-Rust#160](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/160)
+- Update to tonic 0.9  [EventStoreDB-Client-Rust#162](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/162)
```
Comment (YoEight) [inline CHANGELOG.md:13]: You don't need to update this file, it will be done automatically by one of our bots.

**Review by YoEight — CHANGES_REQUESTED** [review]
Thanks again for your contribution!

I do believe it would be safer to regenerate the grpc code. You can achieve that by uncommenting the property `build` in the client Cargo.toml file. When you are done, comment it back.

**Comment by henriiik** [comment]
@YoEight thanks for the feedback! I have regenerated the GRPC code and removed the entry from the changelog.

**Review by YoEight — APPROVED** [review]
Nice work!

**Pull request merged**